subxt/src/tx/tx_client.rs

```diff
@@ -135,12 +135,15 @@ impl<T: Config, C: OfflineClientT<T>> TxClient<T, C> {
     }
 
     /// Creates a signed extrinsic without submitting it.
-    pub fn create_signed_with_nonce<Call, Signer>(
+    ///
+    /// Note: This function will **not** refine the params to use the latest account nonce and the latest block hash/number.
+    /// Instead your call will get default values `nonce = 0` and `mortality = Immortal` for the signed extensions, if the
+    /// params have not been manually set or 'refined' before.
```

Comment:
```suggestion
    /// Note: if not provided, the default account nonce will be set to 0 and the default mortality will be _immortal_.
    /// This is because this method runs offline, and so is unable to fetch the data needed for more appropriate values.
```

As above